signer/migrations/0020__drop_reclaim_script_column.sql

```diff
@@ -0,0 +1,26 @@
+BEGIN;
+-- We want to set reclaim_scipt_hash to NOT NULL, since we dropping
+-- reclaim_script and reclaim_script_hash now will replace the reclaim_script
+-- functionality. In order to do so, we need to delete all rows where 
+-- reclaim_script_hash IS NULL, and rows from another tables pointing via
+-- FOREIGHN KEY to rows where reclaim_script_hash IS NULL.
+
+
+-- Since deposit_signers references deposit requests with ON DELETE CASCADE
+-- this should delete nothing, just added for consistency.
```

Comment:
This should be updated, looks like you removed the ON DELETE CASCADE part of the query.

Reply:
I'd leave this explicit